src/lib/features/change-request-access-service/sql-change-request-access-read-model.ts

```diff
@@ -35,6 +35,23 @@ export class ChangeRequestAccessReadModel
         return !(changeRequestEnabled && !canSkipChangeRequest);
     }
 
+    public async canBypassChangeRequestForProject(
```

Comment:
no check for environment. If a user can skip change requests in any env we're good for now. We can tweak this logic later. For now need something cheap to experiment